src/Microsoft.DiaSymReader.Converter/Utilities/SymReaderHelpers.cs

```diff
@@ -83,7 +83,7 @@ public static ImmutableArray<string> GetImportStrings(ISymUnmanagedReader reader
 
         public static bool TryReadPdbId(PEReader peReader, out BlobContentId id, out int age)
         {
-            var codeViewEntry = peReader.ReadDebugDirectory().FirstOrDefault(entry => entry.Type == DebugDirectoryEntryType.CodeView);
+            var codeViewEntry = peReader.ReadDebugDirectory().LastOrDefault(entry => entry.Type == DebugDirectoryEntryType.CodeView);
```

Comment:
Mike for completeness there is another place you should update from FirstOrDefault to LastOrDefault

There is similar logic in GetPdbPathFromCodeViewEntry in the Pdb2Pdb.cs file (used when infering the PDB file name from the DLL).